### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by sushantdewan123**

### CODE REVIEW

File: src/main/java/com/wavefront/sdk/appagent/jvm/reporter/WavefrontJvmReporter.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+   */
+  public void start() {
+    wfReporter.start(reportingIntervalSeconds, TimeUnit.SECONDS);
+  }
+
+  /**
+   * Stop the reporter. Invoke this method before your JVM shuts down.
+   */
+  public void stop() {
+    heartbeaterService.close();
```
Comment (vikramraman) [inline src/main/java/com/wavefront/sdk/appagent/jvm/reporter/WavefrontJvmReporter.java:58]: Should we also do `wfReporter.stop()`?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+    </distributionManagement>
+
+    <properties>
+        <io.dropwizard.metrics5.version>5.0.0-rc2</io.dropwizard.metrics5.version>
```
Comment (srujann) [inline pom.xml]: This property is not being used in the POM?

File: src/main/java/com/wavefront/sdk/appagent/jvm/reporter/WavefrontJvmReporter.java [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  private final int reportingIntervalSeconds;
+  private final HeartbeaterService heartbeaterService;
+
+  private WavefrontJvmReporter(WavefrontInternalReporter wfReporter,
+                               int reportingIntervalSeconds,
+                               WavefrontMetricSender wavefrontMetricSender,
+                               ApplicationTags applicationTags) {
+    this.wfReporter = wfReporter;
+    this.reportingIntervalSeconds = reportingIntervalSeconds;
+    heartbeaterService = new HeartbeaterService(wavefrontMetricSender, applicationTags,
```
Comment (srujann) [inline src/main/java/com/wavefront/sdk/appagent/jvm/reporter/WavefrontJvmReporter.java:42]: Please update the core sdk version and add source for the heartbeat metric.

**Review by srujann — APPROVED** [review]
LGTM.

**Review by vikramraman — APPROVED** [review]

**Pull request merged**